server/apps/archive/archive_crop.py

```diff
@@ -88,6 +88,11 @@ def _validate_aspect_ratio(self, crop, doc):
         """
         width = doc['CropRight'] - doc['CropLeft']
         height = doc['CropBottom'] - doc['CropTop']
+
+        if width < crop['width'] or width < crop['height']:
```

Comment:
Is the second comparison `width < crop['height']` supposed to be against height or width?